specification/security/resource-manager/Microsoft.Security/preview/2023-09-01-preview/securityConnectorsDevOps.json

```diff
@@ -0,0 +1,3368 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft.Security",
+    "description": "REST APIs for Defender for DevOps",
+    "version": "2023-09-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/securityConnectors/{securityConnectorName}/devops/default/azureDevOpsOrgs": {
+      "get": {
+        "tags": [
+          "default"
+        ],
+        "summary": "Returns a list of Azure DevOps organizations.",
+        "operationId": "List_AzureDevOpsOrg",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
```

Comment:
please always use the latest, current is "v5". please also update other places

Reply:
Updated all references to v5